### PR TITLE
Bugfix for multiple repo pages

### PR DESCRIPTION
**Pull request by mastertinner**
Sorry for already sending you the next pull request but I had to fix a bug in the code I sent you in the last one. Can you please merge it and upgrade to version 1.2.1?

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "strider-gitlab",
-  "version": "1.1.3",
+  "version": "1.2.1",
```
Comment (knownasilya) [inline package.json]: Could you remove this, I try to separate the publishing commits from the actual changes, incase multiple changes have to be made.

**Comment by oliversalzburg** [comment]
You might have to rebase or change your commit so that this can be merged. Either way, the merge conflict needs to be resolved. :)

**Comment by mastertinner** [comment]
Done. Sorry about that, guys... I'm not an experienced PR-er ;-)

**Comment by knownasilya** [comment]
No better time to learn than the present :wink:

**Comment by knownasilya** [comment]
Published to ecosystem and npm.

**Comment by mastertinner** [comment]
:+1: thanks!

**Pull request merged**